### DISCUSSION TITLE
Not all teams available in second stage

### DISCUSSION THREAD

**Discussion by TheKringel** [evroon/bracket#611] [Q&A] [answered]
Hi all,
I installed Bracket and started to configure my first tournament but having an issue to configure the second stage as not all teams are getting displayed.

I have 2 groups with 6 teams in each.
1. Stage has both groups setup as round-robin.
2. Stage I want to configure as follow:
- 1st Group A vs 2nd Group B
- 2nd Group A vs 1st Group B
- 3rd Group A vs 4th Group B
- 4th Group A vs 3rd Group B
- 5th Group A vs 6th Group B
- 6th Group A vs 5th Group B

My problem is, that when I setup a new stage e.g. Round-Robin, I can only select up to 4th team of each group. The 5th and 6th of each team is not in the list.

Is there a reason, why only the first 4 of a group are available in the 2nd stage?

Best regards,
Kringel

**Comment by evroon** [comment]
Hi, thanks for raising this. It's indeed a limitation/rough edge of Bracket for now. I just chose `4` as an arbitrary limit because it worked for me and I didn't want a lot of options in the dropdown:

https://github.com/evroon/bracket/blob/b395d148f0f8899926a30f0f4bb178974403d720/backend/bracket/logic/scheduling/builder.py#L91-L96

We could make it a tournament setting in the frontend or config value or something.

**Reply by TheKringel** [reply]
Thanks for the fast response.
I think it would be required for different scenarios to have the full list available.
As I'm only a technical-user but not a programmer, I'm not really a help to find or suggest the way forward.

From a user-perspective, I don't care if the dropdown-list has 10, 20 or 50 entries. As the list gets anyway filtered once I start typing. But for those who manually scroll through the list and pick the team manually, a long list is indeed inconvenient.

**Reply by evroon** [reply, marked as answer]
Hmm yes maybe having the full list isn't that bad. I usually just scroll, so what you call "manually". Maybe a dropdown is also not really the optimal method of selecting these inputs.

I created an issue to implement this here: https://github.com/evroon/bracket/issues/612

Could you mark this discussion as "Answered"? Then other users will see that the question wasn't unanswered :) The issue will be used for next steps.
